.github/workflows/windows.yml

```diff
@@ -132,7 +113,7 @@ jobs:
 
       - name: CMake configure
         run: |
-          & {{ env.VCVARSPATH }} x64 && cmake -G "Ninja Multi-Config" `
+          & {{ env.VCVARSPATH }} x64 && cmake -G "${{ env.CMAKE_GENERATOR }}" `
             -DENABLE_CPPLINT=OFF `
             -DENABLE_ONEDNN_FOR_GPU=OFF `
```

Comment:
we can remove explicit `ENABLE_ONEDNN_FOR_GPU` because it will be automatically deduced based on `BUILD_SHARED_LIBS`